dune/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -99,12 +103,52 @@ namespace cpgrid
         std::vector<double> coordData;
         ecl_grid.exportCOORD(coordData);
 
-        std::vector<double> zcornData;
-        ecl_grid.exportZCORN(zcornData);
-
         std::vector<int> actnumData;
         ecl_grid.exportACTNUM(actnumData);
 
+        std::vector<double> zcornData;
+        {
+            ecl_grid.exportZCORN(zcornData);
+
+            auto repair = ::Opm::UgGridHelpers::RepairZCORN {
+                std::move(zcornData), actnumData,
+                std::vector<int>{ ecl_grid.getNX() ,
+                                  ecl_grid.getNY() ,
+                                  ecl_grid.getNZ() }
+            };
+
+            zcornData = repair.destructivelyGrabSanitizedValues();
+
+            if (repair.switchedToDepth()) {
+                std::cout << "ZCORN Values Switched from Elevation to "
```

Comment:
> Fine. After the default of that family is to not print those ASCII escape sequences.

I'll add another argument to the setupSimpleDefaultLogging() function for that. I'd like to avoid default arguments though (leaving that for the callers for now).